lib/travis/build/data.rb

```diff
@@ -81,8 +83,12 @@ def env_vars
         data[:env_vars] || []
       end
 
+      def ssh_key?
+        !!ssh_key
```

Comment:
Because `def ssh_key; @ssh_key ||= …`, it seems to me that this method is always going to return `true`. Is that intentional?